### PR TITLE
Changed displayed title

### PR DESCRIPTION
**Pull request by babolivier**
A friend of mine made me notice that it'd be nicer to have the app's display name instead of its slug in the `<title>` tag. For example, having "Cozy - Weather" displayed in the title instead of "Cozy - owm".
If the team agrees with this statement, here is a pull request including this change. :)

### CODE REVIEW

**Comment by babolivier** [comment]
I don't really understand why the tests are failing. If it's because of my changes, just give me an hint so I can give it a try.

**Comment by clochix** [comment]
Hi @babolivier, and thanks for this contribution ☺
Test are broken for some days now, I'm trying to fix them.

Also, our workflow is to merge every contribution to the `development` branch of the applications, so we can test it. We only merge development into `master` before publishing. So, could you please make another PR to the development branch, and without the build? Thanks !

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Vagrant.configure("2") do |config|
```
Comment (clochix) [inline Vagrantfile]: Please don't commit the Vagrantfile. Usually it should not be in your application directory.

**Comment by frankrousseau** [comment]
Hi @babolivier! Thank you for your PR. But there is a problem with it. It comes with many file changes without reason. Can you just keep the changes on the required file?

File: Vagrantfile (hunk 1 above)
Comment (babolivier) [inline Vagrantfile]: Hi,
Roger, I'll open another PR tonight to the `development` branch with only the
related file.

On Mon, Oct 19, 2015 at 10:46 am, Clochix < notifications@github.com [notifications@github.com] > wrote:
In Vagrantfile [https://github.com/cozy/cozy-home/pull/541#discussion_r42347782] :

> @@ -0,0 +1,27 @@
> +Vagrant.configure("2") do |config|


Please don't commit the Vagrantfile. Usually it should not be in your
application directory.

File: client/app/views/main.coffee [hunk 2]
```diff
@@ -234,10 +234,12 @@ module.exports = class HomeView extends BaseView
 
             @selectedApp = slug
 
-            name = @apps.get(slug).get('name')
+            name = @apps.get(slug).get('displayName')
```
Comment (clochix) [inline client/app/views/main.coffee]: Maybe you could write
```
    app = @apps.get slug
    name = app.get('displayName') or app.get('name') or ''
```

Comment (babolivier) [inline client/app/views/main.coffee]: Sure, thanks for the suggestion!

**Comment by babolivier** [comment]
Hi, 
Roger, I'll open another PR tonight to the `development` branch with only the related file.

File: Vagrantfile (hunk 1 above)
Comment (babolivier) [inline Vagrantfile]: Goddammiy, I was sure I ignored this one.

**Pull request closed without merging**